### PR TITLE
Increase default timeout

### PR DESCRIPTION
**Pull request by mazzy89**
Over the weekend Azure experimented a lot of network issues and net-exporter alarms have been triggered all over the Azure installations.

Rather have in place a Confimap and customize the timeout value per TC, it is better to increase it so it will be propagated all over the places and we don't have to do any intervention.

### CODE REVIEW

**Comment by corest** [comment]
I'm not sure about this. `10s` is an extremely large number for timeouts. Keeping this like this just because of Azure makes us blind on issues with other clusters.

**Comment by mazzy89** [comment]
Ok then this would require to customize it per TC. are we on the same page?

**Comment by corest** [comment]
Why don't we check why this timeout is so large?

**Comment by mazzy89** [comment]
Because it is Azure 🤭 who knows why it takes so much time... I mean networking issue in Azure are still a mistery and not easy to debug.

**Comment by corest** [comment]
Well, if that's the way to treat this issue, then let's just drop this alert for azure installation. But that should be discussed further

**Comment by mazzy89** [comment]
I don't agree to drop the alerts in Azure. it's too aggressive approach. we can still have it around and see even if with 10s timeout the net-exporter complains

**Comment by corest** [comment]
`10s` timeout doesn't make sense for me. If we can make it configurable per platform - then we can talk about this. Let's discuss it in @giantswarm/sig-infrastructure meeting

**Comment by ferrandinand** [comment]
I agree with @corest. 
I feel we as SREs, we must agree that giving a 9,9s delay in our network is something we cannot accept as a right service.

If this is a workaround I am fine if we open also a PM/Issue to benchmark or check our networking stack deeper and devote more time to it.

**Comment by mazzy89** [comment]
let me rephrase it. my proposal is to drop this PR and make this value configurable for installation/TC. must not be 10s. can be something smaller. but yeag let's move to @giantswarm/sig-infrastructure

**Comment by ferrandinand** [comment]
That sound fine to me then.

**Comment by mazzy89** [comment]
Moved to inbox of @giantswarm/sig-infrastructure for the moment

**Comment by mazzy89** [comment]
Let's close this and let's validate a hypothesis expressed during the SIG Infra meeting that it to validate whether in Azure there is any SYN flooding protection that can block the traffic
